app/views/admin/applications/_button.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_with url: url, method: method do |f| %>
```

Comment:
This is an interesting design choice because we already have the `button_to` helper, so I'm curious what trade offs you see between this partial you made, and the button_to that comes with Rails?